lang/python26/Portfile

```diff
@@ -61,6 +61,8 @@ configure.ccache        no
 # signed integer overflow is normally undefined as per the C spec, but python
 # needs it to be defined as twos-complement
 configure.cflags-append -fwrapv
+configure.cppflags-append -I${prefix}/include/openssl-1.0
+configure.ldflags-append -L${prefix}/lib/openssl-1.0
```

Comment:
Does this find the right headers and lib with both openssl10 and openssl11 installed? I would have thought it needed to be a prepend.

Reply:
That makes sense, thanks! Changed.

Reply:
Oops, for some unknown reason the latest changes are not commited. Should be correct now.